_docs/upgrade-collator-node.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
@atomaka - here's the starting doc to do the node upgrade. Please do a separate PR against this branch to patch.